source/concepts.tex

```diff
@@ -585,13 +585,8 @@
 \item
   \tcode{S} is \tcode{(void)swap(E1, E2)}\footnote{The name \tcode{swap} is used
   here unqualified.} if \tcode{E1} or \tcode{E2}
-  has class or enumeration type\iref{basic.compound} and that expression is valid, with
-  overload resolution performed in a context that includes the declaration
-\begin{codeblock}
-  template<class T>
-    void swap(T&, T&) = delete;
-\end{codeblock}
-  and does not include a declaration of \tcode{ranges::swap}.
+  has class or enumeration type\iref{basic.compound} and that expression is valid,
+  when `swap` is looked up using only argument-dependent lookup\iref{basic.lookup.argdep}.
```

Comment:
The `\note` is probably unnecessary; we have no such notes in other places.

I'm not happy grammatically with the phrase "X is equivalent to Y, if Y is valid when Z." What we mean is more like "X is equivalent to Y-when-Z, if Y is valid when Z."